src/dioptra/restapi/v1/workflows/schema.py

```diff
@@ -41,3 +43,82 @@ class JobFilesDownloadQueryParametersSchema(Schema):
         by_value=True,
         default=FileTypes.TAR_GZ.value,
     )
+
+
+class SignatureAnalysisSchema(Schema):
+
+    fileContents = fields.String(
+        attribute="file_contents", metadata=dict(description="The contents of the file")
+    )
+
+
+class SignatureAnalysisSignatureParamSchema(Schema):
+    name = fields.String(
+        attribute="name", metadata=dict(description="The name of the parameter")
+    )
+    type = fields.String(
+        attribute="type", metadata=dict(description="The type of the parameter")
+    )
+
+
+class SignatureAnalysisSignatureInputSchema(SignatureAnalysisSignatureParamSchema):
+    required = fields.Boolean(
+        attribute="required",
+        metadata=dict(description="Whether this is a required parameter"),
+    )
+
+
+class SignatureAnalysisSignatureOutputSchema(SignatureAnalysisSignatureParamSchema):
+    """No additional fields."""
```

Comment:
I'm not sure if this will show up somewhere in the Swagger docs, replace with the following just to be safe:

```suggestion
    pass
```